bouquet/assess.py

```
@@ -89,7 +89,7 @@ def relax_structure(atoms: Atoms, energyCalc: Calculator, calc: Calculator, step
         if steps is not None:
             dyn.run(fmax=DEFAULT_FMAX, steps=steps)
         else:
-            dyn.run(fmax=DEFAULT_FMAX)
+            dyn.run(fmax=TIGHT_FMAX)
```

Ruff (0.15.14)

[error] Undefined name `TIGHT_FMAX`

(F821)